treeherder/model/models.py

```diff
@@ -1251,6 +1251,23 @@ def elastic_search_insert(self):
             return es_line
 
 
+class Group(models.Model):
+    """
+    The test harness Group.  Sometimes this is a Manifest, but not always.
+    Not to be confused with JobGroup which is Treeherder specific.
+    """
+    id = models.BigAutoField(primary_key=True)
+    name = models.CharField(max_length=200, unique=True)
```

Comment:
It looks like, with the unique index I've got on that field (because I'm using get_or_create) that the max size I'm allowed is 255:
Reading up in SO: https://stackoverflow.com/a/16474039/333614

"MySQL assumes 3 bytes per utf8 character. 255 characters is the maximum index size you can specify per column, because 256x3=768, which breaks the 767 byte limit."

Indeed, the largest I was able to set it to was 255.

@jgraham said on IRC that the longest path he could find was 178, so perhaps going with 255 will give us the headroom we need.  It's a pretty small table, so far, so the cost is minimal for having a large field length.